axelor-account/src/main/java/com/axelor/apps/account/service/move/MoveTemplateService.java

```diff
@@ -349,24 +355,24 @@ public List<Long> generateMove(LocalDate moveDate, List<HashMap<String, Object>>
                 moveTemplateLine.getAnalyticDistributionTemplate());
             moveLineComputeAnalyticService.generateAnalyticMoveLines(moveLine);
             counter++;
+          } else {
+            taxLineDescription = moveTemplateLine.getName();
           }
         }
+        move.setPartnerBankDetails(
+            move.getMoveLineList().get(0).getPartner().getBankDetailsList().stream()
+                .filter(it -> it.getIsDefault() && it.getActive())
+                .findFirst()
+                .get());
```

Comment:
Please change this code, currently we do not check if any value are null or empty. Be careful, calling `get()` on an empty Optional will return a runtime error.  Instead change the expression to only call move.setPartnerBankDetails if a value is found.

Reply:
Fixed using ifPresent()